Japx/Classes/Core/Japx.swift

```diff
@@ -27,6 +27,63 @@ public enum JapxError: Error {
     case unableToConvertDataToJson(data: Any)
 }
 
+/// `JapxDecodingOptions` is a set of options affecting the decoding of JSON:API into JSON you requested from `Japx.Decoder`.
+public struct JapxDecodingOptions {
+    
+    /// Defines if a relationship that doesn't heve related object stored in `included`
+    /// shoud be parsed as a dictionary of only `type` and `id`.
+    /// If `false` ti will be parsed as `nil`.
+    /// 
+    /// Defaults to false.
+    ///
+    /// - Tag: parseNotIncludedRelationships
+    public var parseNotIncludedRelationships: Bool = false
+    
+    /// Creates an instance with the specified properties.
+    ///
+    /// - parameter parseNotIncludedRelationships: Read more [here](parseNotIncludedRelationships)
+    ///
+    /// - returns: The new `JapxDecodingOptions` instance.
+    public init(parseNotIncludedRelationships: Bool = false) {
+        self.parseNotIncludedRelationships = parseNotIncludedRelationships
+    }
+}
+
+public extension JapxDecodingOptions {
+    
+    /// Default JSON:API to JSON decoding options for `Japx.Decoder`
+    static var `default`: JapxDecodingOptions { .init() }
+}
+
+/// `JapxEncodingOptions` is a set of options affecting the encoding of JSON into JSON:API you requested from `Japx.Encoder`.
+public struct JapxEncodingOptions {
+    
+    /// Common namespace inclued all attribute names, relationship names, `type` and `id`.
+    /// If enabled it will include keyword `meta` into common namepace, expecit the not to
```

Comment:
I'm not sure what did you say here after `into common namepace,` (typo namepace -> namespace), so please reformulate. I'm not sure if you are trying to say that client should not expect to have meta as an attribute or relationship name or something else

Reply:
Updated, please check if it look ok now?